app/models/item.rb

```diff
@@ -8,4 +8,13 @@ class Item < ActiveRecord::Base
   acts_as_taggable_on :diets, :tastes, :ingredients
 
   validates :name, presence: true, length: { maximum: 128 }
+
+  # def self.popular(menu)
+  #   items = menu.items
+  #   pop_array = items.select{|item| item.favorites.count > 0 }.sort_by { |item| item.favorites.count }
+  #   if pop_array.length > 6
+  #     pop_array.slice!(-6, -1)
+  #   end
+  #   pop_array
+  # end
```

Comment:
I wanted to ask the instructors if they knew a better way to return the result via JSON (rather than do it in angular - or if doing it in angular was fine). Will delete after.